_source/_posts/2022-04-07-where-okta-meets-devops.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: blog_post
+title: "How Can DevOps Engineers Use Okta?"
+author: edunham 
+by: advocate
+communities: [devops]
+description: "It's more than just SSO"
+tags: [devops]
+tweets:
+- ""
+- ""
+- ""
+image: blog/where-okta-meets-devops/devops.png
+type: awareness
+---
+
+## What is DevOps? 
+
+DevOps is one of those buzzwords. Ask three DevOps professionals what it means, and you might get four different answers. Most will agree that DevOps usually describes a mix of tools and techniques combining aspects of systems administration, software development, IT, release engineering, security, and often QA. At its best, DevOps synergizes the strengths of these fields to reduce toil through automation and improve software quality through continuous testing and deployment, and [infrastructure as code](https://en.wikipedia.org/wiki/Infrastructure_as_code). At its worst, DevOps is used as an excuse to ask a single under-resourced team to do the work of several groups, in an ever-increasing spiral of tech debt. Most DevOps roles fall somewhere on the spectrum between these extremes. 
+
+While many tech professionals know Okta and its competitors as the way they access their corporate email and chat, there are actually four main areas where Okta's product family interfaces with the DevOps skillset. 
+
+## Accessing SaaS services and tooling
+
+In 2022, it's vanishingly rare for a tech company to host every software product that they use on- premises. Although there are open or [self-hosted versions](https://github.com/awesome-selfhosted/awesome-selfhosted) of every major tool in the modern workflow, maintaining separate services gets prohibitively expensive. Instead of hosting physical servers, most organizations rent cloud resources. Instead of running email ((Mailman)[https://list.org/]), chat ([IRC](https://datatracker.ietf.org/doc/html/rfc1459), [XMPP](https://xmpp.org/about/technology-overview/)), user account management ([LDAP](https://ldap.com/)), and more in-house, companies instead rent services providing these features from specialist providers. 
```

Comment:
```suggestion
In 2022, it's vanishingly rare for a tech company to host every software product that they use on- premises. Although there are open or [self-hosted versions](https://github.com/awesome-selfhosted/awesome-selfhosted) of every major tool in the modern workflow, maintaining separate services gets prohibitively expensive. Instead of hosting physical servers, most organizations rent cloud resources. Instead of running email ([Mailman](https://list.org/)), chat ([IRC](https://datatracker.ietf.org/doc/html/rfc1459), [XMPP](https://xmpp.org/about/technology-overview/)), user account management ([LDAP](https://ldap.com/)), and more in-house, companies instead rent services providing these features from specialist providers. 
```